src/misc.c

```diff
@@ -535,6 +537,16 @@ void putlog EGG_VARARGS_DEF(int, arg1)
   /* Create the timestamp */
   if (shtime) {
     strftime(stamp, sizeof(stamp) - 2, log_ts, t);
+    if ((f = strstr(stamp, "%f"))) { /* handle millisecond specifier %f */
```

Comment:
This is not compatible with the normal strftime syntax where "%%f" would be a literal "%f" and should not be replaced

Reply:
thank you. implemented. ready for review again.